README.md

```diff
@@ -64,41 +68,63 @@ The following is a brief listing of currently supported features:
  * Timeline Recording Track
  * Mesh Import & Export
  * Material Import & Export (USD Preview Surface or DisplayColor)
- * Unity Materials: HDRP, Standard and limited LWRP support
+ * Unity Materials: HDRP, Standard and limited URP support
  * Material Export Plugins
  * Variant Selection
  * Payload Load/Unload
  * Automatic Lightmap UV Unwrapping
  * Skeletal Animation via UsdSkel
  * Scene Instancing
  * Point Instancing
- * Integration with C# Job System
  * High and Low Level Access to USD API via C#
 
+## Importing USD Files
+
+To import a USD file into Unity, use the USD context menu, and select one of the 'Import as ...' options as required
+Supported USD file formats are:
+ * .USD
+ * .USDA
+ * .USDC
+ * .USDZ
+
 ## Importing Materials
 
-To import materials from USD, firstly import the USD file using the USD menu. Then, to get the materials to render in the scene, change the Import Settings > Materials in the Inspector to 'Import Preview Surface'. Finally reimport the USD file.
+To import materials from USD, firstly import the USD file using the USD menu.
+Then, to get the materials to render in the scene, change the 'Import Settings' > 'Materials' in the Inspector to 'Import Preview Surface'. Finally re-import the USD file.
+You can furthermore import all texture files by changing the 'Import Settings' > 'Payload Policy' in the Inspector to 'Load All' and re-import the USD file.
+
+If upon initial import nothing is visible, try re-importing the USD file with the above 'Import Settings'
 
 ## Streaming Playback via Timeline
 
 After importing a USD file with either skeletal or point cache animation, open
-the Timeline window. Select the root of the USD file.
+the Timeline window.
 
-Create a playable director by clicking the "Create" button in the Timeline window.
-Next, drag the root USD file into the Timeline to create a track for this object.
-Finally, drag the USD file once more to add a USD clip to the track for plaback.
-Scrubbing through time will now update the USD scene by streaming dat from USD.
+1. Select the root of the USD file.
+2. Create a playable director by clicking the "Create" button in the Timeline window.
+3. Next, drag the root USD file into the Timeline to create a track for this object.
+4. Finally, drag the USD file once more to add a USD clip to the track for plaback.
+5. Scrubbing through time will now update the USD scene by streaming data from USD.
```

Comment:
This isn't a step, so leave it unnumbered. It doesn't make sense to have another number after step 4 saying 'Finally' :P